Diz.Core/util/ServiceProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using LightInject;
+
+namespace Diz.Core.util
```

Comment:
this is future dependency injection stuff. could be left or commented out for now.